governance/votes/Executive vote - February 22, 2024.md

```diff
@@ -0,0 +1,115 @@
+- Decrease the ETH-B Stability Fee by 0.33 percentage points, from 7.24% to **6.91%**.
+- Decrease the ETH-C Stability Fee by 0.33 percentage points, from 6.49% to **6.16%**.
+- Decrease the WSTETH-A Stability Fee by 0.51 percentage points, from 7.16% to **6.65%**.
+- Decrease the WSTETH-B Stability Fee by 0.51 percentage points, from 6.91% to **6.40%**.
+- Decrease the WBTC-A Stability Fee by 0.02 percentage points, from 6.70% to **6.68%**.
+- Decrease the WBTC-B Stability Fee by 0.02 percentage points, from 7.20% to **7.18%**.
+- Decrease the WBTC-C Stability Fee by 0.02 percentage points, from 6.45% to **6.43%**.
+
+### Spark Protocol Dai Direct Deposit Module (D3M) Parameter Changes
+
+**Authorization:** [Ecosystem Approval](https://forum.makerdao.com/t/feb-9-2024-proposed-changes-to-sparklend-for-upcoming-spell/23656/3)\
+**Proposal:** [Forum post "[Feb 9, 2024] Proposed Changes to SparkLend for Upcoming Spell"](https://forum.makerdao.com/t/feb-9-2024-proposed-changes-to-sparklend-for-upcoming-spell/23656)
+
+If this executive proposal passes, following the recommendation of Ecosystem Actor Phoenix Labs, the following Spark Protocol Dai Direct Deposit Module (D3M) parameters will be changed:
+
+- Increase the DIRECT-SPARK-DAI [Maximum Debt Ceiling (`line`)](https://manual.makerdao.com/module-index/module-dciam#maximum-debt-ceiling-line) by 300 million DAI from 1.2 billion DAI to **1.5 billion DAI**.
+- Increase the DIRECT-SPARK-DAI [Target Available Debt (`gap`)](https://manual.makerdao.com/module-index/module-dciam#target-available-debt-gap) by 20 million DAI from 20 million DAI to **40 million DAI**.
+- Increase the DIRECT-SPARK-DAI [Ceiling Increase Cooldown (`ttl`)](https://manual.makerdao.com/module-index/module-dciam#ceiling-increase-cooldown-ttl) by 12 hours from 12 hours to **24 hours**.
+- Increase the DIRECT-SPARK-DAI buffer by 20 million DAI from 30 million DAI to **50 million DAI**.
+
+### Housekeeping Actions
+
+**Authorization:** [Ecosystem Approval](https://forum.makerdao.com/t/proposed-housekeeping-items-upcoming-executive-spell-2024-02-22/23697/2), [2](http://forum.makerdao.com/t/proposed-housekeeping-items-upcoming-executive-spell-2024-02-22/23697/4)\
+**Proposal:** [Forum post "Proposed Housekeeping items (Upcoming Executive Spell - 2024-02-22)"](https://forum.makerdao.com/t/proposed-housekeeping-items-upcoming-executive-spell-2024-02-22/23697)
+
+#### Push USDP out of the Input Conduit
+
+If this executive proposal passes, following the recommendation of the Governance Facilitators in collaboration with the spell team, the following operations will be performed:
+
+- Raise `PSM-PAX-A` DC to **1,000,000 DAI**
+- Call push() on `MCD_PSM_PAX_A_INPUT_CONDUIT_JAR` (use push(uint256 amt)) to push **754,598.72 USDP**
+- Call void() on `MCD_PSM_PAX_A_JAR`
+- Set `PSM-PAX-A` DC to **0 DAI**
+
+This will have the following net results:
+
+- **Debt Ceiling** remains at **0 DAI** after the transaction has been executed.
```

Comment:
**nit** - inconsistent use of DC vs Debt Ceiling

```suggestion
- Raise `PSM-PAX-A` Debt Ceiling to **1,000,000 DAI**
- Call push() on `MCD_PSM_PAX_A_INPUT_CONDUIT_JAR` (use push(uint256 amt)) to push **754,598.72 USDP**
- Call void() on `MCD_PSM_PAX_A_JAR`
- Set `PSM-PAX-A` Debt Ceiling to **0 DAI**

This will have the following net results:

- **Debt Ceiling** remains at **0 DAI** after the transaction has been executed.
```